### PR TITLE
artifactory-ha existing pvc

### LINKED ISSUE #173: Existing PVC Name in Artifactory-HA is incorrect
Existing PVC Name in values.yaml shows the following:

## If true, you must prepare a PVC with the name e.g `artifactory-ha-member`

We have noticed the "Existing PVC Name" as following:
volume-artifactory-ha-artifactory-ha-primary
volume-artifactory-ha-artifactory-ha-member

This is not Documented anywhere and causes issue when mapping the artifactory-ha deployment to existing claims.

### PR DESCRIPTION
**Pull request by danielezer**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md

**What this PR does / why we need it**:
This PR adds the option to use existing persistent volume claims for Artifactory primary and member nodes

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
fixes #173

### CODE REVIEW

**Comment by danielezer** [comment]
@eldada after discussion with @jainishshah17 we decided to give the option to use existing PVCs by name since choosing PVCs by labels in statefulSets is not supported

**Comment by rimusz** [comment]
@danielezer can resolve the conflict, please

**Comment by danielezer** [comment]
@rimusz done

**Comment by jainishshah17** [comment]
@eldada We are waiting for you to review and merge it.

File: stable/artifactory-ha/README.md [hunk 1]
```diff
@@ -460,6 +481,7 @@ The following table lists the configurable parameters of the artifactory chart a
 | `artifactory.primary.javaOpts.xms`              | Artifactory primary node java Xms size           |                     |
 | `artifactory.primary.javaOpts.xmx`              | Artifactory primary node java Xms size           |                     |
 | `artifactory.primary.javaOpts.other`            | Artifactory primary node additional java options |                     |
+| `artifactory.primary.persistence.existingClaim` | Whether to use an existing pvc for the primary node | `true`            |
```
Comment (eldada) [inline stable/artifactory-ha/README.md]: The default value should be `false`

Comment (eldada) [inline stable/artifactory-ha/README.md]: Need to also add 
```
artifactory.node.persistence.existingClaim
```

Comment (danielezer) [reply stable/artifactory-ha/README.md]: Right, thanks. Nice catch

Comment (danielezer) [reply stable/artifactory-ha/README.md]: Thanks, added

**Pull request merged**